app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1256,16 +1256,28 @@
                             <show_in_website>0</show_in_website>
                             <show_in_store>0</show_in_store>
                         </extensions_compatibility_mode>
-                        <crate_admin_user_notification translate="label comment">
+                        <create_admin_user_notification translate="label comment">
```

Comment:
I would leave the typo as-is. Yes, you move the value to the new setting, but legacy code may still use the old xpath.

Reply:
For legacy code, this can be used, if I remember correctly:
```xml
<crate_admin_user_notification deprecated="true">
    <config_path>admin/security/create_admin_user_notification</config_path>
</crate_admin_user_notification>
```

Reply:
Where is that @luigifab ?

Reply:
Here:

https://github.com/OpenMage/magento-lts/blob/6ea8cf0ae16ed17b7866443fa4a3ecc2292e498e/app/code/core/Mage/Paypal/etc/system.xml#L29-L40